### ISSUE TITLE
Make fat report file

### ISSUE THREAD

**Issue by hypery2k** [galenframework/galen#215]
It would be really helpful if we create if create a single file (HTML or PDF) of the report. This can be useful for sending the reports via email...

**Comment by ishubin** [comment]
Ok, sounds reasonable. At the moment I am doing a big refactoring and changing the way the HTML and JSON reports are generated. Once this is finished we can discuss this story

**Comment by hypery2k** [comment]
I know, I just wanted to mention it, because you're currently redesigning this part ;)
If you need help do not hesitate ;)

**Comment by ishubin** [comment]
sure, thanks :) I am almost finished. Going to push later a big chunk of work.
I like the idea of PDF reports because you can also nicely embed images in it

**Comment by hypery2k** [comment]
we can use [PDFBox](https://pdfbox.apache.org):
```
public void addImageToDocument(final PDDocument doc, final BufferedImage bufferedImage) {
  PDXObjectImage ximage = new PDJpeg(doc, bufferedImage);
}
```

**Comment by hypery2k** [comment]
i searched a little bit for a client side solution. What do you think about http://pdfmake.org/playground.html?

**Comment by ishubin** [comment]
I would prefer to have a more simple solution. This seems to complex for such a minor feature.  And why not use an open-source Java library for that http://pdfbox.apache.org/?

**Comment by hypery2k** [comment]
you're right. Better to use this one. I will try a little bit with the library
